docs/source/notebooks/stochastic_gradient_for_cnn.ipynb

```diff
@@ -151,7 +151,9 @@
     "    # 10 discrete output classes -> pymc3 categorical distribution\n",
     "    out = pm.Categorical('out', prediction, observed=target_var)\n",
```

Comment:
Why not using `total_size` kwarg here? Providing total_size to step method seems to be inconsistent.